### ISSUE TITLE
Incorrect URLs when GLPI site accessed by URL rather than IP address

### ISSUE THREAD

**Issue by yild** [glpi-project/glpi#15006]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.6

### Bug description

On my GLPI server I had created Apache site conf to removed /glpi/ and to access it via URL rather than by IP, while doing so ATM some URLs are not generated properly.

My dashboard URL is located at https://glpi.mydomain.com/front/central.php
While selecting Entity (via "Select the desired entity") address with /glpi/ is still generated  i.e.
https://glpi.mydomain.com/glpi/front/central.php?active_entity=7 - which gives 404 error.

glpi code is located in /var/www/glpi/, in Apache logs there are errors that points to wrong file locations like this

`
[Sun Jun 25 01:27:41.552812 2023] [core:info] [pid 129892] [client 10.10.1.5:17643] AH00128: File does not exist: /var/www/glpi/glpi/front/central.php, referer: https://glpi.mydomain.com/front/config.form.php
[Sun Jun 25 01:27:41.926524 2023] [core:info] [pid 129892] [client 10.10.1.5:17643] AH00128: File does not exist: /var/www/glpi/favicon.ico, referer: https://glpi.mydomain.com/glpi/front/central.php?active_entity=7
`

It seams that $CFG_GLPI['root_doc'] is incorrectly generated, when used in this case in /ajax/entitytreesons.php

My apache site conf:

```
<VirtualHost *:80>

        ServerName glpi.mydomain.com
        ServerAlias glpi.mydomain.com

        RewriteEngine On
        RewriteRule ^(.*)$ https://%{HTTP_HOST}$1 [R=301,L]

</VirtualHost>

<VirtualHost *:443>

        ServerName glpi.mydomain.com
        ServerAlias glpi.mydomain.com
        ServerAdmin webmaster@localhost

        DocumentRoot /var/www/glpi/

        DirectoryIndex index.php index.phtml index.html index.htm

        SSLEngine on
        SSLCertificateFile /etc/ssl/certs/glpi.mydomain.com.selfsigned.crt
        SSLCertificateKeyFile /etc/ssl/private/glpi.mydomain.com.selfsigned.key

        ErrorLog ${APACHE_LOG_DIR}/glpi.error.log
        CustomLog ${APACHE_LOG_DIR}/glpi.access.log combined

</VirtualHost>

```

**Comment by 0xNath** [comment]
Hello, 
have you tried to clean your browser cache after doing the change to your URL ?

If that doesn't work, have you changed your URL in the GLPI configs ?

![image](https://github.com/glpi-project/glpi/assets/67143274/a1a5af7b-4de4-4a4f-9b2c-f1949e0a3dd4)

**Comment by yild** [comment]
Cache was cleared and URL app is set correctly (without /glpi).

**Comment by cedric-anne** [comment]
Hi,

It has been fixed by #14295 in GLPI 10.0.7.
